src/main/java/com/example/travelservice/controllers/TravelerController.java

```diff
@@ -28,4 +28,14 @@ public Traveler createTraveler(@RequestBody TravelerDto travelerDto) {
         Traveler traveler = travelerConverter.convertTraveler(travelerDto);
         return travelerService.createTraveler(traveler);
     }
+
+
+    @PutMapping("traveler/id/{id}/update")
+    public Traveler updateEmail(@PathVariable Long id, @RequestBody TravelerDto travelerDto){
```

Comment:
you need to rename this method, it's not updateEmail, it's just update or updateTraveler

Reply:
fixed